public/expressions/helpers.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getClient } from '../services';
+import {
+  VisLayerTypes,
+  VisLayerError,
+  VisLayerErrorTypes,
+  PointInTimeEventsVisLayer,
+} from '../../../../src/plugins/vis_augmenter/public';
+
+export const getAlerts = async (
+  monitorId: string,
+  startTime: number,
+  endTime: number
+): Promise<string> => {
+  const params = {
+    size: 1000,
+    sortField: 'start_time',
+    sortDirection: 'asc',
+    monitorIds: [monitorId],
+  };
+
+  const resp = await getClient().get('/api/alerting/alerts', { query: params });
+
+  if (resp.ok) {
+    const filteredAlerts = resp.alerts.filter((alert) => alert.start_time >= startTime && alert.start_time <= endTime );
+    return filteredAlerts;
```

Comment:
The return type for this method is `Promise<string>` but this is not a string. So, either we update the return type or change this based on how this is being consumed

Reply:
Created the alert data model and passed it here. We can later use that alert data model and do some refactor in the code base to use that.